frontend/src/components/Field/Rating/Rating.tsx

```diff
@@ -0,0 +1,186 @@
+import { Fragment, useEffect, useMemo, useState } from 'react'
+import {
+  Box,
+  forwardRef,
+  Grid,
+  Stack,
+  Text,
+  useBreakpointValue,
+  Wrap,
+  WrapItem,
+} from '@chakra-ui/react'
+
+import { FieldColorScheme } from '~theme/foundations/colours'
+
+import { RatingOption } from './RatingOption'
+
+export interface RatingProps {
+  /**
+   * The `name` attribute forwarded to each rating `radio` element
+   */
+  name: string
+  /**
+   * Number of rating options to render.
+   */
+  numberOfRatings: number
```

Comment:
would `maxRating` be a more appropriate name?

Reply:
I've thought about that naming, but `maxRating` may cause the user to infer that the component can render less than the maxRatings?

Instead, `numberOfRatings` explicitly declare the number of ratings declared. wdyt?

Reply:
sounds good